examples/swarmion-full-stack/.github/workflows/deploy-staging.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  push:
+    branches:
+      - main
+
+# cancel previous deploy runs
+concurrency:
+  group: main
+  cancel-in-progress: true
+
+name: 🚀 Deploy staging
+
+env:
+  NODE_VERSION: 16
+  AWS_REGION: us-east-1 # update with your region
```

Comment:
Maybe this should be referenced in the documentation then

Reply:
I added a new commit which remove this env variable by using the backend-core provider region defined rather than duplicating it here